Garden.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="gardenstyle.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@500;700&display=swap" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+</head>
+
+<body>
+    <nav class="navbar">
+        <!-- using max width so that it can be applicable on all the devices, -->
+        <div class="max-width">
+            <div class = "box"></div>
+            <div class="logo"><a href="#">Gardener</a></div>
+            <ul class="menu">
+                <li><a href="#home">Home</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#Contact">Contact</a></li>
+            </ul>
+            <div class="menu-btn">
+                <i class="fa fa-bars"></i>
+            </div>
+        </div>
+    </nav>
+
+
+    <!-- home section -->
+    <section class="home" id="home">
+        <div class="max-width">
+            <div class="home-content">
+                <div class="text-1">Welcome to <span>Gardener</span></div>
+                <div class="text-2">Landscaping Company</div>
+                <div class = "text-3"><p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Molestias reprehenderit vitae, ipsam optio blanditiis eum ex perspiciatis expedita odit quisquam aliquam beatae minima dolores eveniet rem, molestiae ut dolore natus nam voluptates! Alias expedita odio delectus corrupti voluptatem reiciendis neque!</p></div>
+                <a href="#">Contact Us </a>
+            </div>
+        </div>
+        <div><img src="C:\Users\Karma Soni\Downloads\images\img-1.jpg" class = "homeimg"></div>
```

Comment:
This will not work for hosted pages. Please add image assets

Reply:
@Karma-1710 unresolved

Reply:
okay, will upload images assets to the GitHub repo